### DISCUSSION TITLE
Save icon shouldn't be the send icon

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#44166] [Mobile] [answered]
Editing a discussion shouldn't show the send icon 🛫, it should show some sort of save icon ✔️ for saving. 

![Screenshot_20230112-190422.png](https://user-images.githubusercontent.com/2119212/212206922-db3e3a18-ce26-47aa-a90c-0130899f14b1.png)

**Comment by stevepopovich** [comment, marked as answer]
Hi @jsoref thanks for the feedback. Passed this onto the team for prioritization
